src/database/DataTypes/VaccineVialMonitorStatusLog.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Sustainable Solutions (NZ) Ltd. 2020
+ * mSupply Mobile
+ */
+
+import Realm from 'realm';
+
+export class VaccineVialMonitorStatusLog extends Realm.Object {}
+
+VaccineVialMonitorStatusLog.schema = {
+  name: 'VaccineVialMonitorStatusLog',
+  primaryKey: 'id',
+  properties: {
+    id: 'string',
+    status: 'VaccineVialMonitorStatus',
+    date: { type: 'date', default: new Date() },
+    itemBatch: 'ItemBatch',
```

Comment:
should this have a `Option`?

Reply:
uh, pardon my ignorance. why? I'm missing something..

Reply:
So that we can see the full 'ledger' or historic vvm status' and when they changed of an item batch, an item batch can have many 'logs', where the most recent one is the current status. So you can see when a vaccine has transitioned from level 1 -> 2 -> 3 -> 4 and when. Make sense? Thoughts?

Reply:
I'm still confused, but I won't comment on a merged PR because that could be confusing in itself.

I get the many 'logs' attached to an item batch and the use of a ledger display. I don't see why `Option` is mentioned or what that should do. but don't worry, I don't think I need to know.

Reply:
ohhh - sorry. And also sorry for replying to you telling me not to reply. I meant should the 'log' have a 'reason' - i.e. make a new category of 'reasons' for vvm status changes so a user could select why the vvm status changed.. i.e. 'fridge exploded', 'left fridge door open oops!' etc..

sorrrrrry 🤣

Reply:
oh, yeah, right. that makes sense - thanks! 
and now I've replied too.. oops! In that case, yes. `Option` would be good